### PR TITLE
Removed zendesk widget from homepage

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
- Fixes https://github.com/mitodl/micromasters/issues/1270

#### What's this PR do?
- Removed zendesk widget from homepage

#### Where should the reviewer start?
- see zendesk_widget.js

@pdpinch @aliceriot @noisecapella 

#### Screenshots (if appropriate)
screen shot 2016-10-08 at 3 34 03 pm

### CODE REVIEW

**Comment by aliceriot** [comment]
I'm not seeing the widget on `/learners` or `/dashboard` anymore, should I be?

**Comment by aliceriot** [comment]
oop nvm that's my browser blocking the request

**Comment by Ferdi** [comment]
Yes, they should be

**Comment by aliceriot** [comment]
functionality looks good!

File: ui/templates/base.html [hunk 1]
```diff
@@ -15,6 +15,9 @@
     <script type="text/javascript" src="{{ style_public_src }}"></script>
     {% endif %}
     <script type="text/javascript" src="{{ style_src }}"></script>
+    {% if zendesk_widget %}
+      <script type="text/javascript" src="{{ zendesk_widget }}"></script>
+    {% endif %}
```
Comment (aliceriot) [inline ui/templates/base.html]: I think this is a cleaner approach overall (compared to the big script tag)

Comment (aliceriot) [inline ui/templates/base.html]: so :100:

**Pull request merged**